Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml

```diff
@@ -432,11 +453,13 @@ tasks:
       {
         "position": {
           "x": 1155,
-          "y": 2910
+          "y": 3600
         }
       }
     note: false
-    timertriggers: []
+    timertriggers:
+    - fieldname: detectionsla
```

Comment:
We are not detection tool so we cant determine the detection SLA

Reply:
Stayed ther by mistake. fixed